admin/client/src/containers/CampaignAdmin/CampaignAdminList.js

```diff
@@ -134,27 +134,31 @@ class CampaignAdminList extends SilverStripeComponent {
       );
     });
 
-    // Get preview details
-    const classNames = itemLinks
-      ? 'cms-content__split cms-content__split--left-sm'
-      : 'cms-content__split cms-content__split--none';
+    // Set body
+    const body = accordionBlocks.length
+      ? (<Accordion>{accordionBlocks}</Accordion>)
+      : (
+        <div className="alert alert-warning" role="alert">
+          <strong>This campaign is empty.</strong> You can add pages by selecting{' '}
+          <em>Add to campaign</em> from within the <em>More Options</em> popup on{' '}
+          the edit page screen.
```

Comment:
If you were to add that link it should be added to the notice rather than the preview.
For the preview message I would either leave the text "no preview available" or remove the text completely.